### ISSUE TITLE
Optional parameters cannot be used

### ISSUE THREAD

**Issue by mnapoli** [PHP-DI/Slim-Bridge#9]
``` php
$app->get('/users[/{userId}]', function ($userId = null) {
});
```

If `userId` is provided it works fine. If not, then PHP-DI fails because it cannot resolve the `$userId` argument.

I think that we simply need to add the [`DefaultValueResolver`](https://github.com/PHP-DI/Invoker/blob/master/src/ParameterResolver/DefaultValueResolver.php) to [the array of resolvers](https://github.com/PHP-DI/Slim-Bridge/blob/master/src/config.php#L56-L61) (as last item).

**Comment by bmwadforth** [comment]
Has this been fixed? I cannot seem to use optional route parameters in Slim 3 with PHP-DI.

**Comment by mnapoli** [comment]
@bmwadforth can you check #37 -> is that your problem?

Else yes this has been fixed in #10, if you see a bug feel free to open an issue with an example of something that doesn't work.
